ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__rosa-4.13-candidate-x86-loaded-upgrade-from-4.12.yaml

```diff
@@ -36,6 +36,7 @@ tests:
     cluster_profile: aws-perfscale-qe
     env:
       CHANNEL_GROUP: candidate
+      CHURN: "false"
```

Comment:
What? Seems unrelated to this PR.

Reply:
Yeah, it's unrelated, new PR?, the fix is pretty obvious